korge/src/commonMain/kotlin/com/soywiz/korge/render/RenderContext.kt

```diff
@@ -498,22 +498,6 @@ class RenderContext constructor(
 
     private val drawTempTexture: AGTexture by lazy { AGTexture() }
 
-    fun drawBitmap(frameBuffer: AGFrameBuffer, bmp: Bitmap) {
-        drawTempTexture.upload(bmp, mipmaps = false)
```

Comment:
It wasn't used.

Reply:
Please, restore this. I guess it should include a test to be referenced, but I think it might make sense to be here.